ci-operator/step-registry/vsphere/provision/bastionhost/vsphere-provision-bastionhost-commands.sh

```diff
@@ -21,6 +21,11 @@ if [[ ! -f "${bastion_ignition_file}" ]]; then
 fi
 bastion_ignition_base64=$(base64 -w0 <"${bastion_ignition_file}")
 
+if [[ -z "${vsphere_connected_portgroup}" ]]; then
```

Comment:
Before this, we might need to add 
```
vsphere_connected_portgroup: ${vsphere_connected_portgroup}"
```

Reply:
Or we can change to 
`if [[ -z "${vsphere_connected_portgroup:-}" ]]; then`
or else if vsphere_connected_portgroup is not defined vsphere_context.sh, got error 
`vsphere_connected_portgroup: unbound variable`

Reply:
change `[[ -z "${vsphere_connected_portgroup}" ]] ` to `[[ -z "${vsphere_connected_portgroup:-}" ]]`
to fix below error 
```
/bin/bash: line 25: vsphere_connected_portgroup: unbound variable
```

Reply:
Changes implemented